src/Microsoft.AspNetCore.Sockets/HttpSocketBuilderExtensions.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Microsoft.Extensions.DependencyInjection;
+
+namespace Microsoft.AspNetCore.Sockets
+{
+    public static class HttpSocketBuilderExtensions
+    {
+        public static ISocketBuilder UseEndPoint<TEndPoint>(this ISocketBuilder socketBuilder) where TEndPoint : EndPoint
```

Comment:
Isn't the convention for terminal middleware to start with "Run" instead of "Use"